tasks/The Fish game.md

```diff
@@ -43,7 +43,15 @@ The weighting between the noise field and the spotlight were manipulated to gene
 Participants had the same role
 
 ## Instructions
-ASKED AND WAITING RESPONSE
+Written by CSS Lab
```

Comment:
```suggestion
> Written by CSS Lab
```

Reply:
Perhaps we should use the same string in every instance of this, so we can easily search for it later etc.